src/shipit_risk_assessment/requirements.txt

```diff
@@ -1 +1,3 @@
--e ./../../lib/cli_common[pulse,taskcluster,mercurial] #egg=mozilla-cli-common
+-e ./../../lib/cli_common[pulse,taskcluster,mercurial]#egg=mozilla-cli-common
```

Comment:
please leave space before ``#``, this way installation works with both nix and with virtualenv+pip